projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -32,6 +32,8 @@ import { ICON_TYPE } from '../../misc/icon/index';
 import { HamburgerMenuService } from './../../../layout/header/hamburger-menu/hamburger-menu.service';
 import { NavigationNode } from './navigation-node.model';
 
+const ARIA_EXPANDED_ATTR = 'aria-expanded';
```

Comment:
SonarQube check asked to move 'aria-expanded' literal to a variable, since its used 3 times in this component